### PR TITLE
Update compare-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Made slight changes to reduce ambiguity.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6516

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Distribution groups are best for situations where you need to broadcast informat
 
 Distribution groups can be [upgraded to Microsoft 365 Groups](../manage/upgrade-distribution-lists.md).
 
-Distribution groups can be added to a team in Microsoft Teams.
+Distribution groups can be added to a team in Microsoft Teams. The members get added in the team and not the group as an object.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
Distribution groups can be added to a team in Microsoft Teams. The members get added to the team and not the group as an object.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**